mcs/build/tests.make

```diff
@@ -157,6 +157,9 @@ $(test_makefrag): $(test_response)
 
 -include $(test_makefrag)
 
+build-test-lib: $(test_lib)
```

Comment:
Because otherwise I have to type the actual expansion of $(test_lib) as the make target when building the test suite from the shell.

Typing `make monodroid_corlib_test.dll` and friends gets old really fast.